extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpHostConfigSource.java

```diff
@@ -45,19 +45,11 @@ public String getValue(String propertyName) {
             // In dev-mode we want to only listen on localhost so others on the network cannot connect to the application.
             // However, in WSL this would result in the application not being accessible,
             // so in that case, we launch it on all interfaces.
-            return (LaunchMode.current().isDevOrTest() && !isWSL()) ? "localhost" : ALL_INTERFACES;
+            return (LaunchMode.current().isDevOrTest() && !LaunchMode.isWSL()) ? "localhost" : ALL_INTERFACES;
```

Comment:
Also not related to your patch but it seems weird we only override `quarkus.http.host` as there are other problematic values as seen in the `VertxHttpRecorder` extract below.

I think it makes sense to default to `0.0.0.0` in the WSL Linux case for most cases. WDYT @cescoffier @radcortez ?

Reply:
I believe so.